components/automate-ui/src/app/pages/project/rules/project-rules.component.html

```diff
@@ -0,0 +1,110 @@
+<chef-page
+  subheading="Resources that match the type and all conditions are included in the project."
+  confirm-btn-text="Save Rule"
+  [attr.heading]="getHeading()"
+  [attr.disable-confirm]="!ruleForm.valid"
+  [attr.page-loading]="isLoading"
+  [attr.confirm-loading]="saving"
+  (confirm)="saveRule()"
+  (close)="closePage()">
+  <div *ngIf="!isLoading">
+    <ng-container>
+      <form id="ruleForm" [formGroup]="ruleForm" (ngSubmit)="saveRule()">
```

Comment:
Hmm... editing a rule to add, delete, or change condition(s) does not work. I can add a second condition to a rule with one condition and save it, but then when re-opening the rule the second condition does not show up. Similarly I can delete a condition from a rule with 4 conditions, but it comes back when I re-open. Same thing with editing a value.
So any kind of condition editing is not being saved.

Oh, probably the back-end wiring is not done, right? So this would be for another card.

Reply:
yup, that should be part of the follow-up work that the auth team does